examples/get-started/react/widgets/app.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {createRoot} from 'react-dom/client';
+import DeckGL, {GeoJsonLayer, ArcLayer} from 'deck.gl';
+import {
+  CompassWidget,
+  ZoomWidget,
+  FullscreenWidget,
+  DarkGlassTheme,
+  LightGlassTheme
+} from '@deck.gl/widgets';
+import '@deck.gl/widgets/stylesheet.css';
+
+/* global window */
+const prefersDarkScheme = window.matchMedia('(prefers-color-scheme: dark)');
```

Comment:
Nit: I wasn't aware that there was a standardized way to query for dark mode. Seems like something that could be built into the widget theming system as a default? `PreferredTheme` or something like that? Though maybe it is better for exposition to show it here.

Reply:
I prefer the exposition here since apps implement their themes in so many different ways. Maybe they don't want the "Glass" version, maybe they let the user override theme in a setting, etc..